### PR TITLE
Update trait documentation as GeneratesShroud was renamed to CreatesShroud

### PR DESCRIPTION
**Pull request by Mailaender**
Followup of https://github.com/OpenRA/OpenRA/pull/8479.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/RevealOnDeath.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public class RevealOnDeathInfo : ConditionalTraitInfo
 		[Desc("Radius of the reveal around this actor.")]
 		public readonly WDist Radius = new WDist(1536);
 
-		[Desc("Can this actor be revealed through shroud generated by the GeneratesShroud trait?")]
+		[Desc("Can this actor be revealed through shroud generated by the CreatesShroud trait?")]
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/RevealOnDeath.cs]: `[Desc("Can this actor be revealed through shroud generated by the " + nameof(CreatesShroud) + "trait?")]` would provide compiler guarantees that this error can't happen again.

Comment (Mailaender) [reply OpenRA.Mods.Common/Traits/RevealOnDeath.cs]: Good idea. I found some more occasions to use it.

**Comment by pchote** [comment]
Was removing the backticks around the trait names (and therefore removing the `highlighting` on the docs page) intentional? I'm not fussed either way, but want to make sure that this wasn't an oversight.

**Comment by teinarss** [comment]
Can we please use the string interpolation syntax here?

**Comment by pchote** [comment]
The compiler won't accept that: strings in attributes must be compile-time state, while interpolation is syntax sugar for runtime substitution 😞

**Comment by teinarss** [comment]
Doh, that right. This is what happens when you try to suggest things when
you are at the gym 🤣

**Comment by Mailaender** [comment]
Brought the backticks back. I thought they weren't rendered on @readthedocs, but I was wrong.

**Review by abcdefg30 — APPROVED** [review]

**Review by pchote — APPROVED** [review]

**Pull request merged**